pygotham/sponsors/__init__.py

```diff
@@ -11,5 +11,5 @@ def get_accepted():
     """Get the accepted sponsors."""
     return Sponsor.query.join(Level).filter(
         Sponsor.accepted == True,
-        Level.event == g.current_event,
+        Level.event == g.current_event,  # NOQA
```

Comment:
Ha. Yeah. I think as long as it's anywhere in the expression it works, but I agree that it should be on the relevant line.